docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -42,38 +40,22 @@ Download the following configuration files:
 
 {% capture lessoncontent %} 
 
-## Create a PersistentVolume
-
-MySQL and Wordpress each use a PersistentVolume to store data. While Kubernetes supports many different [types of PersistentVolumes](/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes), this tutorial covers [hostPath](/docs/concepts/storage/volumes/#hostpath).
+## Create PersistentVolumeClaims and PersistentVolumes
 
-**Note:** If you have a Kubernetes cluster running on Google Kubernetes Engine, please follow [this guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/persistent-disk).
-{: .note}
+MySQL and Wordpress each require a PersistentVolume to store data.  Their PersistentVolumeClaims will be created at the deployment step.
 
-### Setting up a hostPath Volume
+Many cluster environments have a default StorageClass installed.  When a StorageClass is not specified in the PersistentVolumeClaim, the cluster's default StorageClass is used instead.
 
-A `hostPath` mounts a file or directory from the host node’s filesystem into your Pod. 
+When a PersistentVolumeClaim is created, a PersistentVolume is dynamically provisioned based on the StorageClass configuration.
 
-**Warning:** Only use `hostPath` for developing and testing. With hostPath, your data lives on the node the Pod is scheduled onto and does not move between nodes. If a Pod dies and gets scheduled to another node in the cluster, the data is lost. 
+**Warning:** In local clusters, the default StorageClass uses the `hostPath` provisioner.  `hostPath` volumes are only suitable for development and testing. With `hostPath` volumes, your data lives on the node the Pod is scheduled onto and does not move between nodes. If a Pod dies and gets scheduled to another node in the cluster, the data is lost.
 {: .warning}
 
-1. Launch a terminal window in the directory you downloaded the manifest files.
-
-2. Create two PersistentVolumes from the `local-volumes.yaml` file:
-
-       kubectl create -f local-volumes.yaml
-
-{% include code.html language="yaml" file="mysql-wordpress-persistent-volume/local-volumes.yaml" ghlink="/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/local-volumes.yaml" %}
-
-{:start="3"} 
-3. Run the following command to verify that two 20GiB PersistentVolumes are available:
-
-       kubectl get pv
-
-   The response should be like this:
+**Note:** If you are bringing up a cluster that needs to use the `hostPath` provisioner, the `--enable-hostpath-provisioner` flag must be set in the `controller-manager` component.
```

Comment:
It's not trivial to set controller-manager flags. Is this set by default?

Reply:
It is not set by default because hostpath provisioners are only meant for development/local clusters and should not be used in production.

If you use local-up.sh, there is a parameter that you can pass that will set the flag for you.  But I'm not sure if we should be prescribing any specific cluster deployment methods in this tutorial.